doc/bridge_packages/flame_tiled/flame_tiled.md

```diff
@@ -3,6 +3,42 @@
 **flame_tiled** is the bridge package that connects the flame game engine to [Tiled] maps by parsing
 TMX (XML) files and accessing the tiles, objects, and everything in there.
 
+To use this,
+
+1. Create your own map by [Tiled].
+2. Create a component and add it to the component tree as follows.
+
+```dart
+final component = await TiledComponent.load(
+  'my_map.tmx',
+  Vector2.all(32),
+);
+
+add(component);
+```
+
+
+## Limitations
+
+
+### Flip
+
+[Tiled] has a flip feature that flips tile to horizontally
+or vertically or even rotates it. It's also available in **flame_tiled** too.
+But there is a caution to use this. It's fine with the normal map.
+But note that it would drop the performance if you have a big texture.
+The big texture means you have dozens of tilesets(or a huge tileset)
+and the sum of their dimension is thousands.
+Fortunately, you can set `ignoreFlip = true` and it disables all the flips in the map.
```

Comment:
In our docs the line length is max 100, so you don't have to line-break earlier, so this block of text just needs some realignment.

Reply:
But Markdown-lint complains as `doc/bridge_packages/flame_tiled/flame_tiled.md:30:101 MD013/line-length Line length [Expected: 100; Actual: 130]`. Doesn't it matters?

Reply:
Yeah, it should be less than or equals to 100, this lining warning says it is 130 long.

Reply:
You should have line breaks, but the line breaks should be as close too 100 characters as possible without going over it. Just like we have 80 as max length for code, but for markdown we have 100.

Reply:
Ah~ I got it. Thanks for the explanation :)